### DISCUSSION TITLE
How to run the elrsV3. lua script file in Ethos

### DISCUSSION THREAD

**Discussion by wienne** [FrSkyRC/ETHOS-Feedback-Community#3283] [General]
Upgrade the ELRS firmware of Vanta's external TX to 3.3.2,

How to run the elrsV3. lua script file in Ethos

升级Vanta外置TX 的 ELRS固件到3.3.2，
在Ethos里怎么运行elrsV3.lua脚本文件

**Comment by bsongis-frsky** [comment]
elrsV3.lua is a script for OpenTX
Some dedicated scripts have been developed for Ethos, they are in this repo, in the lua directory

**Comment by wienne** [comment]
试过了，
目录 https://github.com/FrSkyRC/ETHOS-Feedback-Community/tree/main/lua/elrs 
mail.lua 文件
在Ethos里运行，没有任何内容输出。
